tests/components/deconz/test_lock.py

```diff
@@ -66,11 +66,14 @@ async def test_locks(hass):
     gateway.api.event_handler(state_changed_event)
     await hass.async_block_till_done()
 
-    door_lock = hass.states.get("lock.door_lock")
-    assert door_lock.state == STATE_LOCKED
+    assert hass.states.get("lock.door_lock").state == STATE_LOCKED
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)